{
    "config": {
        "abort": {
            "already_configured": "A conta j\u00e1 foi configurada",
            "reauth_failed_existing": "N\u00e3o foi poss\u00edvel atualizar a entrada de configura\u00e7\u00e3o. Remova a integra\u00e7\u00e3o e configure-a novamente.",
            "reauth_successful": "A reautentica\u00e7\u00e3o foi bem-sucedida",
            "unknown": "Erro inesperado"
        },
        "error": {
            "cannot_connect": "Falha ao conectar",
            "invalid_api_key": "Chave de API inv\u00e1lida",
            "not_main_key": "Tipo de chave de API incorreto detectado, use a chave de API 'principal'",
            "reauth_failed_matching_account": "A chave de API fornecida n\u00e3o corresponde ao ID da conta da configura\u00e7\u00e3o existente.",
            "unknown": "Erro inesperado"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "Chave da API"
                },
                "description": "Voc\u00ea precisa fornecer uma nova chave de API 'principal' do UptimeRobot",
                "title": "A autentica\u00e7\u00e3o expirada para {name}"
            },
            "user": {
                "data": {
                    "api_key": "Chave da API"
                },
                "description": "Voc\u00ea precisa fornecer a chave de API 'principal' do UptimeRobot"
            }
        }
    },
    "entity": {
        "sensor": {
            "monitor_status": {
                "state": {
                    "down": "Baixo",
                    "not_checked_yet": "Ainda n\u00e3o verificado",
                    "pause": "Pausa",
                    "seems_down": "Parece estar para baixo",
                    "up": "Para cima"
                }
            }
        }
    }
}